backend/core/utils/__init__.py

```diff
@@ -503,3 +527,20 @@ def get_ip_version(prefix):
     if ":" in prefix:
         return "v6"
     return "v4"
+
+
+def hijack_log_field_formatter(hijack_dict):
+    logged_hijack_dict = {}
+    try:
+        fields_to_log = set(hijack_dict.keys()).intersection(HIJACK_LOG_FIELDS)
+        for field in fields_to_log:
+            logged_hijack_dict[field] = hijack_dict[field]
+        # instead of storing in redis, simply add the hijack url upon logging
+        if "hijack_url" in HIJACK_LOG_FIELDS and "key" in hijack_dict:
+            logged_hijack_dict["hijack_url"] = "https://{}/main/hijack?key={}".format(
```

Comment:
this will return an IP though not a DNS name right? it might not work if it's internal network etc

Reply:
and how can we extract the proper IP address of DNS name?

Reply:
Maybe we can simply add a new variable that reads: ARTEMIS_WEB_HOST especially for the backend?

Reply:
I added ARTEMIS_WEB_HOST, could you re-check now?